middleware/password.recovery.js

```diff
@@ -1,4 +1,57 @@
 const express = require('express');
 const app = express();
 const nodemailer = require('nodemailer');
+const dotenv = require("promise-dotenv").config()
```

Comment:
you can change this to dotenv instead of promise-dotenv. 
We agreed on using the dotenv